sss/add_city.py

```diff
@@ -0,0 +1,56 @@
+import glob
```

Comment:
Let's name these files just `city.py`, `puma.py` and `report.py`. The "add" isn't really adding anything 😉

Reply:
Fixed. Change the name to the city, puma and report